src/renderer/ui/components/modals/AlarmModal.js

```diff
@@ -81,6 +81,7 @@ class AlarmModal extends Component {
       >
         <TimePicker
           hintText={TranslationProvider.query('button-text-click-here-to-set-alarm')}
+          textFieldStyle={{width: '100%'}}
```

Comment:
Sorry, it`s my very first time with github, node, npm and of course material-ui. Can i make such minor cosmetic pull requests in future?